package.json

```diff
@@ -66,50 +66,50 @@
   "devDependencies": {
     "@types/node": "^9.4.6",
     "JSONStream": "^1.3.2",
-    "autocannon": "^2.0.0",
+    "autocannon": "^2.3.0",
     "branch-comparer": "^0.4.0",
     "concurrently": "^3.5.1",
     "cors": "^2.8.4",
     "coveralls": "^3.0.0",
     "dns-prefetch-control": "^0.1.0",
     "fast-json-body": "^1.1.0",
-    "fastify-plugin": "^1.0.0",
+    "fastify-plugin": "^1.0.1",
     "frameguard": "^3.0.0",
     "h2url": "^0.1.2",
-    "helmet": "^3.10.0",
+    "helmet": "^3.12.0",
     "hide-powered-by": "^1.0.0",
     "hsts": "^2.1.0",
-    "http-errors": "^1.6.2",
+    "http-errors": "^1.6.3",
     "ienoopen": "^1.0.0",
-    "joi": "~11.3.4",
+    "joi": "~13.2.0",
```

Comment:
Does this work with the way we use Joi? That ecosystem went all-in on `async`/`await`.

Reply:
Unfortunately it supports node 8+.
I’m -1 to update.